packages/app-bridge/src/react/usePageTemplateSettings.spec.ts

```diff
@@ -0,0 +1,90 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { cleanup, renderHook, waitFor } from '@testing-library/react';
+import { afterEach, describe, expect, it } from 'vitest';
+import { getAppBridgeThemeStub } from '../tests';
+import { usePageTemplateSettings } from './usePageTemplateSettings';
+
+const HUB_API_SETTINGS_GET = {
```

Comment:
Why not create a dummy for these?

Reply:
Hub API shouldn't be used anymore (thus the `@deprecated` on all the methods). So to avoid any usage of `HubDummy` I just didn't create it